src/wmnds/patterns/travel-updates/_travel-updates.njk

```diff
@@ -0,0 +1,280 @@
+                description: "Grand Central - St Chad’s",
+                severityClass: "wmnds-disruption-indicator-medium--error",
+                statusIcon: "general-warning-triangle",
+                status: "Major disruption"
+            }
+        ]
+    }
+] %}
+
+{# Start travel updates content card #}
+{% call wmndsContentCard({ classes: "wmnds-content-card--travel-updates"}) %}
+    <div class="wmnds-p-sm wmnds-col-1">
+    {# Show personal style if personal or edit view is true #}
+    {% if isPersonalView or isEditView  %}
+        <h2 class="wmnds-content-card__title">My travel updates</h2>
+        {# Loop through personal travel updates #}
+        {% for update in myTravelUpdates %}
+        <div class="wmnds-travel-update wmnds-travel-update--personal{{editViewClass}}">
+            {# Show toggle if not in edit view #}
+            {% if isEditView === false %}
+            
+            <div class="wmnds-travel-update__disruption-title">
+                <h3 class="wmnds-m-none">{{update.title}}</h3>
+                <button class="wmnds-travel-update__disruption-detail-toggle" data-show-details="false">
+                    {{
+                        wmndsIcon({
+                            icon: "general-chevron-right",
+                            title: "Toggle details"
+                        })
+                    }}
+                </button>
+            </div>
+            {# Else just show the title #}
+            {% else %}
+            <h3 class="wmnds-m-b-md">{{update.title}}</h3>
+            {% endif %}
+
+            {# Loop through disruptions #}
+            {% for disruption in update.disruptions %}
+            <div class="wmnds-travel-update__disruption">
+                {# Display disruption indicator #}
+                <button class="wmnds-travel-update__disruption-indicator-btn" aria-expanded="false">
+                    {{
+                        mediumDisruption({
+                            iconLeft: "modes-isolated-" + update.mode,
+                            iconRight: null if isEditView else disruption.statusIcon,
+                            contentText: disruption.service,
+                            capitalize: true if disruption.capitalize else false,
+                            classes: null if isEditView else disruption.severityClass
+                        })
+                    }}
+                </button>
+
+                {% if isEditView === false %}
+                {# Show description and status text/link #}
+                <div class="wmnds-travel-update__disruption-text">
+                    {% if disruption.description %}
+                    <strong>{{disruption.description}}</strong>
+                    {% endif %}
+                    {% if disruption.status === "Good service"%}
+                    <strong>{{disruption.status}}</strong>
+                    {% else %}
+                    {{
+                        wmndsLink({
```

Comment:
Double check all links in this file and make sure the href="#".

If you check your personalised view via https://deploy-preview-609--wmnetwork.netlify.app/patterns/travel-updates/#personalised-view and then you click any of the links within the example, you get directed to a new 
page within the iframe. 

Set to # so the page doesn't change...

![image](https://user-images.githubusercontent.com/13015201/105862565-d3c63b00-5fe7-11eb-8dcf-85907fa7bccf.png)

Reply:
Linked to disruptions.wmnetwork.co.uk in a new tab.